raiden_contracts/tests/fixtures/token.py

```diff
@@ -4,11 +4,19 @@
     CONTRACT_CUSTOM_TOKEN,
 )
 
-token_args = (10 ** 26, 18, CONTRACT_CUSTOM_TOKEN, 'TKN')
+
+@pytest.fixture(scope='session')
+def custom_token_total_supply():
```

Comment:
Why not use plain variables instead of fixtures for constants?

Reply:
I'll try making this a constant in this PR.